odl/util/utility.py

```diff
@@ -417,6 +417,36 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl='general'):
+    """Compute the signal-to-noise ratio.
+    This compute::
```

Comment:
This needs to be fleshed out with more information. See [the documentation on how to document](http://odl.readthedocs.io/dev/document.html) for examples and style.